engine/api/workflow/gorp_model.go

```diff
@@ -58,6 +61,7 @@ func init() {
 	gorpmapping.Register(gorpmapping.New(NodeRun{}, "workflow_node_run", true, "id"))
 	gorpmapping.Register(gorpmapping.New(sqlNodeRun{}, "workflow_node_run", true, "id"))
 	gorpmapping.Register(gorpmapping.New(JobRun{}, "workflow_node_run_job", true, "id"))
+	gorpmapping.Register(gorpmapping.New(JobRunInfo{}, "workflow_node_run_job_info", true, "id"))
```

Comment:
i think this is useless because you don't use to insert/select/update or delete in table workflow_node_run_job_info